### PR TITLE
Add Serial LED Driver to Cuav CAN node GPS (This also fixes the pin map)

### PR DESCRIPTION
**Pull request by davids5**
Dependant on https://github.com/PX4/NuttX/pull/129 coming in.

### CODE REVIEW

File: platforms/nuttx/src/px4/common/srgbled/srgbled.cpp [hunk 1]
```diff
@@ -0,0 +1,166 @@
...
+
+typedef union {
+	uint8_t  grb[3];
+	uint32_t l;
+} led_data_t;
+
+#if defined(BOARD_HAS_N_S_RGB_LED) && !defined(S_RGB_LED_DMA)
+
+
+int led_pwm_servo_set(unsigned channel, uint8_t  value);
```
Comment (dagar) [inline platforms/nuttx/src/px4/common/srgbled/srgbled.cpp:101]: I'm not sure I follow. You implement empty stubs for the led_pwm API, then do all the actual work in a new function (led_pwm_servo_set_ex) that isn't part of that API? Why not just create a new frontend driver for the neopixel?

Comment (davids5) [reply platforms/nuttx/src/px4/common/srgbled/srgbled.cpp:101]: Yes. I pushed so you could get your part working. 
This was done just to test the new physical driver code (scopeing the data) with working RGB code. The NeoPixel driver should have an addressable LED interface (as supported in the LED API) and the low level will have to support that as well. But this should get us started.

**Comment by LorenzMeier** [comment]
@davids5 Is the right NuttX submodule merged?

**Comment by davids5** [comment]
@dagar - neopixal added. Testing off PWM removed
@LorenzMeier - ready

**Review by LorenzMeier — APPROVED** [review]

**Pull request merged**